lib/max_subarray.rb

```diff
@@ -2,7 +2,18 @@
 # Time Complexity: ?
 # Space Complexity: ?
 def max_sub_array(nums)
```

Comment:
👍 Time/space complexity?

Reply:
sorry about that 😓
Time Complexity: O(n)
Space Complexity: O(n)